ansible/memcached-operator/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/ansible-operator:v0.12.0
```

Comment:
Missing update ansible Redme with

```
- [docker][docker_tool] version 17.03+
- [kubectl][kubectl_tool] v1.14.1+
- [operator_sdk][operator_install]
- Access to a Kubernetes v1.14.1+ cluster
```

Reply:
https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/user-guide.md I was following above version requirements. Do you still want me to  update the versions?

Reply:
|o| we need to update the versions across. 
See that kube just tests with k8s upper 1.14.1 
And then, I am quite sure that the current k8s API used is no longer compatible with `Access to a Kubernetes v.1.9.0+ cluster.`

However, it shows out of the context  anyway.. so we can do something across all repos place.s